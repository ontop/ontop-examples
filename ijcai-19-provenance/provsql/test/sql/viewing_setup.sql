\set ECHO none
SET search_path TO public, provsql;

SELECT create_provenance_mapping('d', 'personnel', 'name');

SELECT value FROM d ORDER BY value;
