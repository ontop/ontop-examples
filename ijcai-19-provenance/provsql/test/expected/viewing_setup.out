\set ECHO none
 create_provenance_mapping 
---------------------------
 
(1 row)

  value   
----------
 Dave
 Ellen
 John
 Magdalen
 Nancy
 Paul
 Susan
(7 rows)

